TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/InductionChangeReason.cshtml

```diff
@@ -8,11 +8,67 @@
     <govuk-back-link data-testid="back-link" href="@Model.BackLink">Back</govuk-back-link>
 }
 
-<h1 data-testid="page-title" class="govuk-heading-l">@ViewBag.Title</h1>
-
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
-        <form action="@LinkGenerator.InductionChangeReason(Model.PersonId, Model.JourneyInstance!.InstanceId)" method="post">
+        <form action="@LinkGenerator.InductionChangeReason(Model.PersonId, Model.JourneyInstance!.InstanceId)" method="post" enctype="multipart/form-data">
+            <span class="govuk-caption-l">Induction - @Model.PersonName</span>
+            <govuk-radios asp-for="ChangeReason" data-testid="reason-choices">
```

Comment:
nitpicking but we're using the suffix -options rather than -choices elsewhere for radio buttons